src/serde.rs

```diff
@@ -94,7 +94,7 @@ pub mod float {
 ///
 /// #[derive(Serialize, Deserialize)]
 /// pub struct StringExample {
-///     #[serde(with = "rust_decimal::serde::string")]
+///     #[serde(with = "rust_decimal::serde::str")]
```

Comment:
CI is not currently testing the `serde-with-str` feature

Reply:
Oh, interesting. This is the documentation tests in particular... I wonder if this would've been picked up in publish?

I may see if we can capture this. Less blind spots the better!